numpy/random/mtrand/mtrand.pyx

```diff
@@ -4731,6 +4736,8 @@ cdef class RandomState:
 
         k           = len(alpha)
         alpha_arr   = <ndarray>PyArray_ContiguousFromObject(alpha, NPY_DOUBLE, 1, 1)
+        if np.any(np.less_equal(alpha_arr, 0)):
```

Comment:
Out of curiosity, is there something in the documentation or some other resource that promotes using `less_equal`, `logical_not`, etc. functions over the operators (the above can also be written `np.any(alpha_arr <= 0)`), or leads to believe these functions are the only way to do things like this?